coalahtml/coala_html.py

```diff
@@ -81,8 +81,12 @@ def main():
                 sys.exit(1)
 
         Handler = http.server.SimpleHTTPRequestHandler
+        socketserver.TCPServer.allow_reuse_address = True
```

Comment:
Probably. Because `allow_reuse_address` is a class variable https://hg.python.org/cpython/file/2.7/Lib/SocketServer.py#l408 :D